### PR TITLE
Runs collecstatic and compress managment commands

### PR DESCRIPTION
**Pull request by sayalilunkad**
* Runs collecstatic command to collect static file
  in single location
* Adds COMPRESS_OFFLINE option to local_settings.py
* Runs compress command to compress content outside
  of request/response cycle

### CODE REVIEW

File: chef/cookbooks/horizon/recipes/server.rb [hunk 1]
```diff
@@ -258,6 +258,28 @@
...
 
+execute "python manage.py collectstatic" do
+  cwd dashboard_path
+  environment(PYTHONPATH: dashboard_path)
+  command "python manage.py collectstatic --noinput"
+  user node[:apache][:user]
+  group node[:apache][:group]
+  action :nothing
+  subscribes :run, "template[#{local_settings}]", :immediately
+  notifies :restart, resources(service: "apache2"), :immediately
```
Comment (tboerger) [inline chef/cookbooks/horizon/recipes/server.rb]: I don't think that you want to restart the webserver, in worst case reload should be enough, even if I don't know why you need that.

File: chef/cookbooks/horizon/recipes/server.rb [hunk 2]
```diff
@@ -258,6 +258,28 @@
...
+
+execute "python manage.py compress" do
+  cwd dashboard_path
+  environment(PYTHONPATH: dashboard_path)
+  command "python manage.py compress"
+  user node[:apache][:user]
+  group node[:apache][:group]
+  action :nothing
+  subscribes :run, "template[#{local_settings}]", :immediately
+  notifies :restart, resources(service: "apache2"), :immediately
```
Comment (tboerger) [inline chef/cookbooks/horizon/recipes/server.rb]: I don't think that you want to restart the webserver, in worst case reload should be enough, even if I don't know why you need that.

File: chef/cookbooks/horizon/recipes/server.rb [hunk 3]
```diff
@@ -258,6 +258,28 @@
...
 end
 
+execute "python manage.py collectstatic" do
+  cwd dashboard_path
+  environment(PYTHONPATH: dashboard_path)
+  command "python manage.py collectstatic --noinput"
+  user node[:apache][:user]
+  group node[:apache][:group]
+  action :nothing
+  subscribes :run, "template[#{local_settings}]", :immediately
```
Comment (tboerger) [inline chef/cookbooks/horizon/recipes/server.rb]: Why do you need `:immediately`?

File: chef/cookbooks/horizon/recipes/server.rb [hunk 4]
```diff
@@ -258,6 +258,28 @@
...
+end
+
+execute "python manage.py compress" do
+  cwd dashboard_path
+  environment(PYTHONPATH: dashboard_path)
+  command "python manage.py compress"
+  user node[:apache][:user]
+  group node[:apache][:group]
+  action :nothing
+  subscribes :run, "template[#{local_settings}]", :immediately
```
Comment (tboerger) [inline chef/cookbooks/horizon/recipes/server.rb]: Why do you need `:immediately`?

**Comment by sayalilunkad** [comment]
@tboerger thank you for the review. Made the changes accordingly. Excuse my ignorance with ruby please ;D

**Comment by tboerger** [comment]
LGTM

**Comment by rsalevsky** [comment]
+1

**Comment by dirkmueller** [comment]
Why was this merged? only one +1...

**Comment by toabctl** [comment]
I wanted to ask the same...

**Comment by aspiers** [comment]
Presumably because we didn't decide as a team whether LGTM means +1 or not.

**Comment by toabctl** [comment]
@aspiers "+1" is "+1". "LGTM" is "LGTM". "I love this commit" means "I love this commit". "This is beautiful" means "This is beautiful". We decided that we need 2x "+1". Not 2x LGTM, not 2x "This is beautiful" :-)
But we may need to clarify that during the next workshop.

**Pull request merged**